doc/release/README.multilocale

```diff
@@ -88,10 +89,19 @@ To run Chapel on multiple locales, the following steps are required:
            to use normal ssh-agent/ssh-add capabilities to support
            password-less ssh-ing.
 
-           Note that it is possible to configure GASNet to launch jobs
-           with SLURM using the following command:
+           Also note that if you are using SSH to launch jobs, you might get a
+           login banner printed out along with your program's output. We have
+           found the following setting useful to disable such printing:
+
+             export GASNET_SSH_OPTIONS="-o LogLevel=Error"
```

Comment:
FYI, it would probably also work to set GASNET_SSH_OPTIONS="-q", but one of us should confirm this before putting it in the doc, so let's leave it as a someday-todo.